android/src/paper/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt

```diff
@@ -0,0 +1,16 @@
+package com.swmansion.rnscreens
+
+import android.view.ViewGroup
+import com.facebook.react.bridge.ReactContext
+import androidx.annotation.UiThread
+import com.facebook.react.uimanager.PixelUtil
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.bridge.WritableNativeMap
+import kotlin.math.abs
+
+abstract class FabricEnabledViewGroup constructor(context: ReactContext?) : ViewGroup(context) {
+  protected fun updateScreenSizeFabric(width: Int, height: Int) {
+    // do nothing
```

Comment:
Maybe let's change the function to updateScreenSize?
And then on paper add this implementation and on fabric the fabric version?